keyboards/hacro_8/readme.md

```diff
@@ -0,0 +1,16 @@
+# Hacro 8
+
+The Hacro 8 is a 2x4 mechanical keyboard with a WS2812B under each key and 2 on the underside of the PCB. It uses a 32u4 with the caterina bootloader. 
+
+![Hacro_8](https://lh3.googleusercontent.com/LNV81nz8awpURCpk4A7lmaNVFCsmUr5_6B89qjRqX6CINRyT_DtHuhOacxQhUmwG_yohhmFlp7uv9_winY--B47fbbK9FZU8lJVO4XMCdA6r870o2OsuiUWXWPJw_8Dqqr21ZvpTPg=w2400)
+![Hacro_8_PCB](https://lh3.googleusercontent.com/GP4x0EBqyVq7chv3KFF3O3yM9DodspuB1AkPtNVFsCw5458kvMqt_hVDdPlkAO88aDI6_0YyIneCESLbFnoHbDu7aWJVtY5qBIAf6m6MTNweYyVo30dJSXAs30CbDbWLKhaIvYNkoQ=w2400)
+
+* Keyboard Maintainer: Martyn Evans
+* Hardware Supported: Hacro 8
+* Hardware Availability: etsy link
```

Comment:
Maybe add a link here?

Reply:
No link available at this time. I've moved on to a different project, so I'll have to update this when I get back to working on the keyboard. Cheers